frontend/lib/api/categories.ts

```diff
@@ -0,0 +1,13 @@
+import { z } from "zod";
```

Comment:
more of a general comment. Seeing that we are still using zustand for state management for auth. Do we want to migrate this to redux so it is more compatible with RTK?

Reply:
Realizing this might be something you're still working ^^